src/app/modal/components/lose-changes/lose-changes.component.html

```diff
@@ -0,0 +1,14 @@
+<div mat-dialog-content lose-changes>
+    <h1 class="govuk-heading-m">Unsaved changes</h1>
+    <span class="govuk-hint">
+      If you leave this page, your changes will be lost.
+    </span>
+  </div>
+
+  <button id="test-lose-changes" class="govuk-button" data-module="govuk-button" (click)="onLoseChangesClick()" cdkFocusInitial>
+    Leave and lose changes
+  </button>
+
+  <a  id= "test-cancel" class="cancel" (click)="onCancelClick()">
+    <span>Continue changing record</span>
+  </a>
```

Comment:
this will work with some tweaks @m-ujaffer. we can play with the govs margin or padding to look accurate. Hence we wont need the custom css file

`<span
            id="test-cancel"
            class="govuk-!-font-size-19 govuk-!-margin-left-5"
          >
            <a
              href="javascript:void(0)"
              class="govuk-link govuk-link--no-visited-state"
              (click)="onCancelClick()"
            >
              <span>Continue changing record</span>
            </a>
          </span>`